File: statsmodels/regression/scrap/testll/glmm_binom_example2.py
import urllib.request
from io import StringIO

import statsmodels.api as sm
import numpy as np
import pandas as pd

# Read in data
url = "http://www.ats.ucla.edu/stat/data/hdp.csv"
httpstream = urllib.request.urlopen(url)
html = httpstream.read().decode('utf-8')
f = StringIO(html)
hdp = pd.read_csv(f)

# Generate GLMM model
fam = sm.families.Binomial()
glm_model = sm.MixedGLM.from_formula("remission ~ IL6", hdp, groups=hdp["DID"],family=fam)

# Fit GLMM model
x0=[0,0,1]
fit = glm_model.fit(start_params = x0)

File: statsmodels/regression/scrap/testll/glmm_binom_example2.R
require(lme4)

hdp <- read.csv("http://www.ats.ucla.edu/stat/data/hdp.csv")
hdp <- within(hdp, {
  Married <- factor(Married, levels = 0:1, labels = c("no", "yes"))
  DID <- factor(DID)
  HID <- factor(HID)
})

m <- glmer(remission ~ IL6 + (1 | DID), data = hdp, family = binomial, nAGQ = 1)
print(logLik(m))